src/api/cpp/cvc5.h

```diff
@@ -2958,6 +2963,9 @@ class CVC5_EXPORT Statistics
     bool d_showDefault = false;
   };
 
+  /** Creates an empty statistics object. */
+  Statistics() = default;
```

Comment:
Why is this needed? Can you include an explanation in the documentation? It is clear why an empty constructor is needed, but not why it needs to be called from the cpp api.

Reply:
A python statistics object needs to own the a `api::Statistics` objects. I haven't found a way to properly construct it (by copy-constructing from `Solver::getStatistics()`). Instead the `Statistics.cstats` member is first default-constructed and then copy-assigned to. Furthermore, this is also subject to visibility on the C++ side, so I can't make it private.